client/src/app/projects/components/project-filters/project-filters.component.html

```diff
@@ -0,0 +1,64 @@
+<form [formGroup]="filtersForm">
+  <mat-form-field class="search-bar">
+    <mat-icon matPrefix>search</mat-icon>
+    <input matInput placeholder="Search Projects" formControlName="search">
+    <button mat-button matSuffix mat-icon-button aria-label="Clear">
+      <mat-icon>close</mat-icon>
+    </button>
+  </mat-form-field>
+
+  <mat-form-field>
+    <mat-select placeholder="Search Type"
+                class="search-type"
+                formControlName="searchType">
+      <mat-option value="AllKeywords">All Keywords</mat-option>
+      <mat-option value="AnyKeyword">Any Keyword</mat-option>
+      <mat-option value="ExactMatch">Exact Match</mat-option>
+    </mat-select>
+  </mat-form-field>
+
+  <mat-form-field>
+    <mat-select placeholder="Filter by status" formControlName="wranglingStatus">
+      <mat-option>None</mat-option>
+      <mat-option *ngFor="let state of wranglingStates" [value]="transformWranglingState(state)">{{state}}</mat-option>
+    </mat-select>
+  </mat-form-field>
+
+  <mat-form-field>
+    <mat-select placeholder="Filter by wrangler" formControlName="wrangler">
+      <mat-option>None</mat-option>
+      <mat-option *ngFor="let wrangler of wranglers$ | async" [value]="wrangler.id">{{wrangler.name}}</mat-option>
+    </mat-select>
+  </mat-form-field>
+
+  <mat-form-field>
+    <mat-select placeholder="Filter by priority" formControlName="priority">
+      <mat-option>None</mat-option>
+      <mat-option *ngFor="let priority of [1,2,3]" [value]="priority">{{priority}}</mat-option>
+    </mat-select>
+  </mat-form-field>
+
+  <mat-form-field>
+    <mat-select placeholder="HCA Publication" formControlName="hcaPublication">
+      <mat-option>None</mat-option>
+      <mat-option [value]="true">Yes</mat-option>
+      <mat-option [value]="false">No</mat-option>
+    </mat-select>
+  </mat-form-field>
+
+  <mat-form-field class="search-bar">
```

Comment:
We can start with a regular input, but this could also be a slider, or a combo.
Also, how would this translate to the query? Will we show projects with cell count not smaller than the given count?

Reply:
I tried with the slider from MAT, it looks a bit ugly with the rest of the inputs IMO. Maybe could be okay with some design and tweaking. 

Yes, I think project with cell count >= to the specified count makes sense @gabsie @Wkt8 @ESapenaVentura @ami-day